src/v2/spec.yaml

```diff
@@ -4974,7 +5057,7 @@ paths:
           $ref: "#/components/responses/400Response"
   /farcaster/feed/channels:
     get:
-      summary: Fetch feed based on channel ids
+      summary: Channel IDs
```

Comment:
Maybe `By channel IDs` since we did "By Eth or Sol address" and similar for users? Ideal to be consistent across different APIs. We should sweat the details, that's where the magic happens :)